src/content/learn/react-compiler.md

```diff
@@ -347,7 +347,7 @@ React Compiler can verify many of the Rules of React statically, and will safely
 
 ### How do I know my components have been optimized? {/*how-do-i-know-my-components-have-been-optimized*/}
 
-[React Devtools](/learn/react-developer-tools) (v5.0+) has built-in support for React Compiler and will display a "Memo ✨" badge next to components that have been optimized by the compiler.
+[React Devtools](/learn/react-developer-tools) (v5.0+) has built-in support for React Compiler and will display a "Memo ✨" badge next to components that have been optimized by the compiler. In React Native, this feature is available from version 0.75.0 onwards, since React Native bundles its own DevTools backend.
```

Comment:
i think we should remove the "5.0" thing and instead refer to the new React Native DevTools (default since 0.76)